modules/pxp-module-puppet

```diff
@@ -32,13 +32,19 @@ env_fix_up = nil
 # use this way of defining the `get_env_fix_up` method so that it can "see"
 # the `env_fix_up` variable
 self.class.send(:define_method, :get_env_fix_up) do
+  # If running in a C or POSIX locale, ask Puppet to use UTF-8
+  base_env = {}
+  if Encoding.default_external == Encoding::US_ASCII
```

Comment:
I think isolating this to environments where Ruby has set the external encoding to `Encoding::US_ASCII` seems reasonable...